canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -98,7 +98,7 @@ $toolbar-divider-width: 1px;
 .toolbar-left-bar {
 	flex-shrink: 5000000; // Make sure left bar always shrinks first
 	display: block;
-	padding-right: 42px;
+	padding-right: $spacing-08;
```

Comment:
Can you double check what this change looks like?

Reply:
For this one, I didn't notice any difference